src/Corax/IndexSearcher/IndexSearcher.cs

```diff
@@ -21,6 +21,7 @@ namespace Corax;
 
 public sealed unsafe partial class IndexSearcher : IDisposable
 {
+    private List<string> _fieldsInIndex;
```

Comment:
Does it make sense to cache this? This is an instance that lives for a single tx, no?

Reply:
Well, somebody can write a query with dynamics and use raw entries view. In such a case, this code will be executed twice. I've deleted this cache because I think it's a super rare case.